content/slice/language-guide/exception.md

```diff
@@ -117,66 +127,41 @@ exception DerivedException : BaseException {
 ```csharp
 public partial class BaseException : SliceException
 {
-    public int ErrorCode;
+    public int ErrorCode { get; set; }
 
-    // Primary constructor
-    public BaseException(
-        int errorCode,
-        string? message = null,
-        System.Exception? innerException = null)
-        : base(message, innerException)
+    public BaseException()
+    {
+    }
+
+    public BaseException(int errorCode)
     {
-        ...
+        this.ErrorCode = errorCode;
     }
 }
 
 public partial class DerivedException : BaseException
 {
-    public double Measurement;
+    public double Measurement { get; set; }
+
+    public DerivedException()
+    {
+    }
 
-    // Primary constructor.
     public DerivedException(
         int errorCode,
-        double measurement,
-        string? message = null,
-        System.Exception? innerException = null)
-        : base(errorCode, message, innerException)
+        double measurement)
+        : base(errorCode)
     {
-        ...
+        this.Measurement = measurement;
     }
 }
 ```
 
 {% /aside %}
 
-### ConvertToInternalError
-
-When the generated code decodes an exception from a payload, it sets the exception's
-[ConvertToInternalError][convert-to-internal-error] property to `true`.
-
-This way, when the implementation of an operation makes an invocation and this invocation throws an exception, by
-default, this exception is not re-sent as-is but gets converted into a response with status code
-[InternalError]. If you don't want this conversion, you need to catch the exception and set
-`ConvertToInternalError` to `false`:
-
-```csharp
-try
-{
-    ...make invocation...
-}
-catch (DispatchException exception)
```

Comment:
What is the current behavior. Unless I misunderstand we're still doing this in the Thermostat example. 

https://github.com/icerpc/icerpc-csharp/blob/b1be5453abe350afaff5338fc3e2b7eaf151bab4/examples/slice/Thermostat/Server/ThermoFacade.cs#L43-L44